_rules/SC2-4-2-page-has-title.md

```diff
@@ -22,19 +22,19 @@ authors:
 
 ### Applicability
 
-The rule applies to any page where the root element is an `html` element, and which is not embedded in another page.
+The rule applies to any page where the [document element](https://www.w3.org/TR/dom/#document-element) is an `html` element, and where the page is not embedded in another page.
 
-**Note**: Pages may be embedded inside other pages through elements such as iframes and object elements.
+**Note**: Pages may be embedded inside other pages through elements such as `iframe` and `object` elements.
 
 ### Expectation 1
 
-The page contains at least one `title` element.
+The [document element](https://www.w3.org/TR/dom/#document-element) has at least one [descendant](https://www.w3.org/TR/dom41/#concept-tree-descendant) that is an HTML `title` element.
 
-**Note**: The `title` element exists in other namespaces such as SVG. These are not `title` elements for HTML document and should be ignored.
+**Note**: The `title` element exists in other namespaces such as SVG. These are not HTML `title` elements and should be ignored for this rule.
 
 ### Expectation 2
 
-The first `title` element contains [non-empty text](#non-empty).
+The [child text content](https://www.w3.org/TR/html/infrastructure.html#child-text-content) of the first HTML `title` element that is a [descendant](https://www.w3.org/TR/dom41/#concept-tree-descendant) of the [document element](https://www.w3.org/TR/dom/#document-element) does not only consist of [Unicode separator characters](https://www.unicode.org/versions/Unicode11.0.0/ch04.pdf#G134153).
```

Comment:
Please use plain language here. We can replace "non-empty" as a definition, but we shouldn't be inlining complex definitions like this. That makes this very difficult to use for less technical people.

Reply:
See https://github.com/auto-wcag/auto-wcag/pull/380#discussion_r270832677